catalog/plugin/catalog-plugin-pdf/src/main/java/ddf/catalog/plugin/pdf/thumbnail/PdfThumbnailPlugin.java

```diff
@@ -0,0 +1,57 @@
+package ddf.catalog.plugin.pdf.thumbnail;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.transform.pdf.PdfThumbnailGenerator;
+import ddf.content.operation.UpdateResponse;
+import ddf.content.operation.impl.UpdateResponseImpl;
+import ddf.content.plugin.ContentPlugin;
+import ddf.content.plugin.PluginExecutionException;
+import ddf.content.plugin.PostUpdateStoragePlugin;
+
+public class PdfThumbnailPlugin implements PostUpdateStoragePlugin {
```

Comment:
The main reason video thumbnail creation was done in the plugins and not in the transformer is that the stored video file is necessary to create the thumbnail. In this case, I don't believe a PostCreate/PostUpdateStoragePlugin is even necessary. `CatalogContentPlugin` will call the appropriate `InputTransformer` on creates and updates: https://github.com/codice/ddf/blob/master/catalog/content/core/content-core-catalogerplugin/src/main/java/ddf/content/plugin/cataloger/CatalogContentPlugin.java#L254. So if the thumbnail is generated in the `InputTransformer`, a storage plugin shouldn't even be necessary.